### ISSUE TITLE
Symbian OS/N-Gage simulator：EKA2L1

### ISSUE THREAD

**Issue by Shimmer2** [libretro/RetroArch#15184]
I wonder if retroarch can integrate the Symbian OS/N Gage simulator: EKA2L1? At present, I see that the author mainly focuses on the Windows platform. Would it be more convenient to be compatible with multiple platforms if ported to Retroarch?
不知retroarch能否整合Symbian OS/N-Gage模拟器：EKA2L1？目前看到作者主要是侧重于windows平台，如果移植到retroarch是不是可以更加方便的兼容多平台？

**Comment by Bradbee** [comment]
yes, it would be able to be in retroarch because its open source.


Just Note im not an admin or mod

**Comment by Bradbee** [comment]
i think that retroarch mods are not gonna see this post for a long time so if you want this core script it using the libretro api

**Comment by gouchi** [comment]
Upstream [issue](https://github.com/EKA2L1/EKA2L1/issues/409).

**Comment by Gabrielcnnunes** [comment]
Will it have an EkA2L1 and J2ME core or not?
